src/pages/learn/index_page.cr

```diff
@@ -0,0 +1,189 @@
+class Learn::IndexPage < PageLayout
+  def hero_content
+    render_hero_content
+  end
+
+  def content
+    div class: "md:-mt-12 md:rounded-lg bg-gray-200 overflow-hidden shadow divide-y divide-gray-200 sm:divide-y-0 sm:grid sm:grid-cols-2 sm:gap-px" do
+      div class: "md:rounded-tl-lg md:rounded-tr-lg sm:rounded-tr-none relative group bg-white p-6 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-500" do
+        div do
+          span class: "rounded-lg inline-flex p-3 bg-green-50 text-green-700 ring-4 ring-white" do
+            tag "svg", class: "h-6 w-6", fill: "none", stroke: "currentColor", viewBox: "0 0 24 24", xmlns: "http://www.w3.org/2000/svg" do
+              tag "path", d: "M9 12h6m-6 4h6m2 5H7a2 2 0 01-2-2V5a2 2 0 012-2h5.586a1 1 0 01.707.293l5.414 5.414a1 1 0 01.293.707V19a2 2 0 01-2 2z", stroke_linecap: "round", stroke_linejoin: "round", stroke_width: "2"
```

Comment:
SVG icons provided by https://heroicons.com/

Reply:
Would it be worth it to abstract these icons to methods so this page is easier to read? The ⬈ in particular since it's repeated so many times.